scripts/detect_compiler_linux.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```

Comment:
This is not a detection script, it's an installing script, so please change the name accordingly